src/main/groovy/me/rexim/morganey/gradle/MorganeyPlugin.groovy

```diff
@@ -2,16 +2,24 @@ package me.rexim.morganey.gradle
 
 import org.gradle.api.Plugin
 import org.gradle.api.Project
+import org.gradle.api.plugins.JavaPlugin
+import org.gradle.api.tasks.bundling.Jar
 
 /**
  * Created by thoma on 03/10/2016.
  */
-class MorganeyPlugin implements Plugin<Project>{
+class MorganeyPlugin implements Plugin<Project> {
 
     @Override
     void apply(Project project) {
-        project.task('hello') << {
+        project.plugins.apply(JavaPlugin.class)
+        project.task('hello') << { // This is to be used for testing purposes only
             println "Hello from the Morganey Plugin"
         }
+        project.task([type: Jar.class], 'package-module') {
+            from('src/main/morganey/package')
```

Comment:
In my issue definition, I meant `package` as a just generic module name. It could as well be `me.rexim` or `me.fornever`, and should anyways create the corresponding directory structure.

I suggest to remove `/package` part here. Will it work without it?